test/unittests/api_test.cpp

```diff
@@ -47,7 +47,7 @@ TEST(api, execution_result_value)
     const ExecutionResult result = Value{1234_u32};
     EXPECT_FALSE(result.trapped);
     EXPECT_TRUE(result.has_value);
-    EXPECT_EQ(result.value.i64, 1234_u32);
+    EXPECT_EQ(result.value.i32, 1234_u32);
```

Comment:
Notpick, we do not actually need `_u32` here, right?

Reply:
No. But using `_u32` gives higher confidence (i.e. someone has thought what the result type is expected). Still gray area though.